tutorial/charts/guestbook/README.md

```diff
@@ -0,0 +1,54 @@
+# guestbook: an example chart for educational purpose.
```

Comment:
Similar comment to one of the previous one - are we planning to move helm101/charts/guestbook/
under tutorial? or tutorial is having a new chart? It would be nice to keep a single chart of guestbook that as such can be used with tutorial. Thanks!

Reply:
Sure, now integrated into helm101/charts